pynicotine/gtkgui/userbrowse.py

```diff
@@ -881,7 +880,7 @@ def on_folder_open_manager_accelerator(self, *args):
             self.on_file_manager()
 
         elif len(self.file_store) >= 1:  # [user is not self]
-            self.on_download_directory_to()
+            self.on_download_directory_to()  # same as Ctrl+Enter
```

Comment:
Ctrl+Enter has a recursive case (which I'm not sure is necessary, tbh), this does not. Also, it feels wrong to have this code in a function named "on_folder_open_manager_accelerator". Shouldn't shortcuts be clearly separated, and not trigger shortcuts further down the chain?

Reply:
> Ctrl+Enter has a recursive case (which I'm not sure is necessary, tbh), this does not

I see your point. The only reason why Ctrl+Enter has a recursive case is for when there are no files then transferring a folder would lead to an unhandled error where nothing happens after selecting a destination folder in the dialog. I think it is better that something always occurs, and this seems like a logical course of action, although admittedly it feels a bit confusing so it is not fully intuitive.

> it feels wrong

I know it seems wrong here, but falling back after returning False doesn't work. An alternative would be to trigger "on_folder_transfer_to_accelerator" which duplicates the condition.

> Shouldn't shortcuts be clearly separated?

Yes, but these aren't what I would call "shortcuts"... these are what I consider to be key "bindings", and my intention is overlap them and provide many alternatives so that it doesn't matter if the user doesn't know how the application's interface is supposed to be operated, they can just mash the keyboard and end up with something useful. In our case, the most useful thing would ideally be a file, or failing that then information about the file.

The downside of this approach is code spaghetti.